SPECS/xapi-nbd.spec

```diff
@@ -57,6 +60,7 @@ make test
 %changelog
 * Wed Jul 01 2020 Samuel Verschelde <stormi-xcp@ylix.fr> - 1.11.0-1.1
 - Rebase on CH 8.2
+- add xapi-nbd-1.11.0-adapt-vbd.create-call.XCP-ng.patch to respect new API
```

Comment:
I'm not sure "respect" works here in English the same that it does in French

Reply:
changed to `adapt to`